staging_manifest.yml

```diff
@@ -2,17 +2,18 @@
 applications:
 - name: federalist-staging
 buildpack: https://github.com/cloudfoundry/nodejs-buildpack#v1.5.4
+stack: cflinuxfs2
 domain: fr.cloud.gov
 disk_quota: 2G
-memory: 512MB
+memory: 1G
```

Comment:
Which was being caused by the `staging_manifest.yml` not configuring the production environment properly. This was causing `npm install` to install all the dependencies like `node-sass` which were crashing the system with an out-of-memory error.